pkg/dcgmexporter/parser.go

```diff
@@ -103,19 +103,19 @@ func extractCounters(records [][]string, c *Config) ([]Counter, []Counter, error
 		}
 
 		if len(record) != 3 {
-			return nil, nil, fmt.Errorf("Malformed CSV record, failed to parse line %d (`%v`), expected 3 fields", i, record)
+			return nil, nil, fmt.Errorf("malformed CSV record, failed to parse line %d (`%v`), expected 3 fields", i, record)
 		}
 
```

Comment:
We have to start the error message with a capital letter for consistency with the remaining error messages.  

The current error messages are not Golang idiomatic, but at least they are consistent.  I recommend reverting changes and correcting all error messages to the Golang convention in a separate PR later as a separate change.

Reply:
Let's discuss this offline.

Reply:
Reverted to original format in favor of a new PR to make error messages consistent.